docs/en/core-services/Caching.md

```diff
@@ -204,70 +190,99 @@ A typical configuration will have three tiers for Memory, SSD, and HDD.
+- Eviction
+- Dynamic Block Placement
 
-Alluxio uses evictors to decide which blocks to remove from Alluxio storage.
-Users can specify different Alluxio evictors to control the eviction process better.
+The eviction, that happens in-line with writes, will attempt to remove blocks based on the order enforced by the block annotation policy.
+The last block in annotated order is the first candidate for eviction regardless of which tier it's sitting on.
 
-Out-of-the-box evictor implementations include:
+Out-of-the-box annotation implementations include:
 
-- **LRUEvictor**: Evicts the least-recently-used blocks until the required space is freed.
-**This is Alluxio's default evictor**.
-- **GreedyEvictor**: Evicts arbitrary blocks until the required space is freed.
-- **LRFUEvictor**: Evicts blocks based on least-recently-used and least-frequently-used with a
+- **LRUAnnotator**: Annotates the blocks based on least-recently-used order.
+**This is Alluxio's default annotator**.
+- **LRFUAnnotator**: Annotates the blocks based on least-recently-used and least-frequently-used orders with a
 configurable weight.
     - If the weight is completely biased toward least-recently-used, the behavior will be the same
-    as the LRUEvictor.
-    - The applicable configuration properties are `alluxio.worker.evictor.lrfu.step.factor` and
-    `alluxio.worker.evictor.lrfu.attenuation.factor` which can be found in
-    [configuration properties]({{ '/en/reference/Properties-List.html' | relativize_url }}#alluxio.worker.evictor.lrfu.step.factor)
-- **PartialLRUEvictor** : Evicts based on a least-recently-used policy but will choose one configured
-storage directory on the local worker with maximum free space and only evict from that directory.
-
-The evictor utilized by workers is determined by the Alluxio property
-[`alluxio.worker.evictor.class`]({{ '/en/reference/Properties-List.html' | relativize_url }}#alluxio.worker.evictor.class).
+    as the LRUAnnotator.
+    - The applicable configuration properties are `alluxio.worker.block.annotator.lrfu.step.factor` and
+    `alluxio.worker.block.annotator.lrfu.attenuation.factor`.
+
+The annotator utilized by workers is determined by the Alluxio property
+[`alluxio.worker.block.annotator.class`]({{ '/en/reference/Properties-List.html' | relativize_url }}#alluxio.worker.block.annotator.class).
 The property should specify the fully qualified class name within the configuration. Currently
 available options are:
 
-- `alluxio.worker.block.evictor.LRUEvictor`
-- `alluxio.worker.block.evictor.GreedyEvictor`
-- `alluxio.worker.block.evictor.LRFUEvictor`
-- `alluxio.worker.block.evictor.PartialLRUEvictor`
+- `alluxio.worker.block.annotator.LRUAnnotator`
+- `alluxio.worker.block.annotator.LRFUAnnotator`
+
+#### Evictor Emulation
+The old eviction policies are now deprecated and Alluxio provided implementations are replaced with appropriate annotation policies.
```

Comment:
In the past, when we deprecate a feature, it is still usable but not recommended any more. 
In this case an exception will be thrown, I would say "the old eviction policies are now removed" instead of "deprecated".